### ISSUE TITLE
module declares its path as: github.com/sirupsen/logrus but was required as: github.com/Sirupsen/logrus

### ISSUE THREAD

**Issue by hedengfeng** [sirupsen/logrus#1041]
github.com/Sirupsen/logrus: github.com/Sirupsen/logrus@v1.4.2: parsing go.mod:
        module declares its path as: github.com/sirupsen/logrus
                but was required as: github.com/Sirupsen/logrus

**Comment by freeformz** [comment]
`github.com/Sirupsen/logrus` is the old path.

Anything using it should be updated to import and require `github.com/sirupsen/logrus`.

If this is in another open source project I'd be happy to PR an update to it for you. Otherwise it's something you/the developers of the software will need to do.

Thanks

**Comment by dolmen** [comment]
Unfortunately my PR #859 that could help to diagnose those issues is for now ignored.

**Comment by Baha-sk** [comment]
It seems this error is back from the dead somehow, I get this error when doing `go mod tidy` on my module:
```
...
        github.com/opencontainers/runc/libcontainer/cgroups imports
        github.com/opencontainers/runc/libcontainer/configs imports
        github.com/Sirupsen/logrus: github.com/Sirupsen/logrus@v1.7.0: parsing go.mod:
        module declares its path as: github.com/sirupsen/logrus
                but was required as: github.com/Sirupsen/logrus
```
 
I stumbled on this issue from [pingcap](https://github.com/pingcap/tidb/issues/22123) while googling the error.

I did look up `github.com/opencontainers/runc/libcontainer/configs` source code on GitHub, and they're using `sirupsen` import correctly 
in both [configs](https://github.com/opencontainers/runc/blob/master/libcontainer/configs/config.go#L13) imports and [go.mod](https://github.com/opencontainers/runc/blame/master/go.mod#L20).

Anything changed to the library recently?

**Comment by yasushi-saito** [comment]
As for github.com/opencontainers/runc, the last published version, v0.1.1, refers to Sirupsen. Try a newer version.

```
go get github.com/opencontainers/runc/libcontainer/configs@v1.0.0-rc93
```

**Comment by mjaggard** [comment]
@freeformz Thanks for your kind offer. Could you provide the required PR(s?) for this error please?

```
go: github.com/rancher/migration-tools/cmd imports
	github.com/kubernetes/kompose/pkg/app imports
	github.com/kubernetes/kompose/pkg/transformer/openshift imports
	github.com/openshift/origin/pkg/image/api imports
	github.com/docker/distribution/manifest/schema1 imports
	github.com/Sirupsen/logrus: github.com/Sirupsen/logrus@v1.8.1: parsing go.mod:
	module declares its path as: github.com/sirupsen/logrus
	        but was required as: github.com/Sirupsen/logrus
```
